.gitignore

```diff
@@ -41,6 +41,7 @@ typings/
 
 # Optional npm cache directory
 .npm
+.yarn/
```

Comment:
This should actually be replaced with:

```ignore
# ignores for Yarn v3 w/o PNP
.pnp.*
.yarn/*
!.yarn/patches
!.yarn/plugins
!.yarn/releases
!.yarn/sdks
!.yarn/versions
```

Among other things, the actual code for newer versions of Yarn is in `.yarn/releases` and should be included. Also, there should be a modification to the root `package.json` to include `packageManager: ...`